dspace-api/src/main/java/org/dspace/content/RelationshipServiceImpl.java

```diff
@@ -342,6 +359,71 @@ public void delete(Context context, Relationship relationship) throws SQLExcepti
         }
     }
 
+    /**
+     * Converts virtual metadata from RelationshipMetadataValue objects to actual item metadata.
+     *
+     * @param context           The relevant DSpace context
+     * @param relationship      The relationship containing the left and right items
+     * @param copyToLeftItem    The boolean indicating whether we want to write to left item or not
+     * @param copyToRightItem   The boolean indicating whether we want to write to right item or not
+     */
+    private void copyMetadataValues(Context context, Relationship relationship, boolean copyToLeftItem,
```

Comment:
This PR depends on PR2577? - https://github.com/DSpace/DSpace/pull/2577/files#diff-2e89fe18482e7ac43a1a4e901a8105b7R370

Reply:
Didn't read the description. Sorry!